### DISCUSSION TITLE
About using modpack

### DISCUSSION THREAD

**Discussion by doemty** [UltimatChamp/optimum-fuse#23] [Q&A]
Maybe a silly question but can I use your modpack as base for mine? I do not sure will I post it on Modrinth or CurseForge, but if I do, I can leave link for your?

**Comment by UltimatChamp** [comment]
Ofc, but if you ever publish it to Modrinth/Curseforge make sure it is licensed under Apache License 2.0 

> I can leave link for your?

Yes

**Reply by doemty** [reply]
Thanks! If I somewhen release my own modpack I will type you
